src/fondant/pipeline.py

```diff
@@ -153,6 +155,132 @@ def from_registry(
             ephemeral_storage_size=ephemeral_storage_size,
         )
 
+    @staticmethod
+    def get_image_manifest(image_ref: str):
+        """Retrieve the Docker image manifest.
+
+        Args:
+            image_ref: The Docker image reference (e.g., 'registry/image:tag').
+
+        Returns:
+            dict: The parsed JSON manifest.
+        """
+        cmd = ["docker", "manifest", "inspect", image_ref]
```

Comment:
I guess you need access to the registry to run this code ?

Reply:
I think it should be fine for public images